app/models/step_run.rb

```diff
@@ -349,6 +377,11 @@ def release_info
     slice(:build_version, :build_number, :updated_at, :platform)
   end
 
+  def sync_store_status!
```

Comment:
Should this happen in a job? This, like some of our other actions, could be really slow. However, not that important since rare case.